approval_polls/templates/approval_polls/edit.html

```diff
@@ -119,15 +119,15 @@ <h4 class='panel-title'>
               <div class='radio'>
                 <label class='full-width'><input type='radio' name='radio-poll-type' {% if poll.vtype == 3 %} checked='true' {% endif %} value='3'>Only specific Email owners (will be invited by Email).
                   <div id='email-input' hidden>
-                    <h6>Enter comma-separated Email IDs:</h6>
+                    <h6>Enter a comma after each email address:</h6>
```

Comment:
Sure, feel free to change it in the other places too. I have raised https://github.com/electology/approval_frame/issues/141 that one of us could look into to fix it if possible.